docs/contributing.md

```diff
@@ -50,6 +52,7 @@ Because this project exports a library that will be used by other projects, it i
   - New src/components files should include changes to storybook
   - Package dependency changes should include `yarn.lock` updates and `yarn audit` outputs
 - All [Jest tests](https://jestjs.io/) will be run in CI and must pass before the branch can be merged
+- [Happo tests](https://docs.happo.io/docs/reviewing-diffs) will be run in CI and all diffs must be approved before the branch can be merged. Navigate to the happo link for instructions on how to review and approve diffs.
```

Comment:
```suggestion
- [Happo visual regression tests](https://docs.happo.io/docs/reviewing-diffs) will be run in CI and all diffs must be approved before the branch can be merged. Navigate to the happo link for instructions on how to review and approve diffs.
```

added suggestion for clarification on what Happo is. Also, do users need admin access to Happo in order to approve/reject changes? That might be a good thing to include.

Reply:
You don't need admin privileges to approve/reject changes. I believe you just need account access. I provided access to the truss.works domain. So anyone working at truss with a truss gmail account will be able to approve/reject changes. Admin on the account have the rights to change account access and similar functions. Also,
_"Apart from the users listed below, Github users who have access to the repository connected to this account also have access."_